packages/material-ui/src/ListItemSecondaryAction/ListItemSecondaryAction.test.js

```diff
@@ -19,4 +23,26 @@ describe('<ListItemSecondaryAction />', () => {
     refInstanceof: window.HTMLDivElement,
     skip: ['componentProp'],
   }));
+
+  it('should render without classes that disable gutters', () => {
+    const { getByRole } = render(
+      <ListItem>
+        <ListItemSecondaryAction />
+      </ListItem>,
+    );
+    const listItem = getByRole('listitem');
+    expect(listItem.querySelector(`div.${classes.disableGutters}`)).to.equal(null);
```

Comment:
Is the div important here? We might change the tag and then the test will always pass regardless of the class logic.

If you want to make sure that `ListItemSecondaryAction` does not have or has the class you can put a `data-testid` on it (`<ListItemSecondaryAction data-testid="secondary-action" />`) and then assert on it: `expect(getByTestId('secondary-action')).to.have.class(classes.disableGutters)`

Reply:
Thanks.  The div is an implementation detail and I shouldn't have done it that way.  I'll make the change.

Reply:
All set.  Thanks again.